src/Grids/vertically_stretched_rectilinear_grid.jl

```diff
@@ -209,11 +209,19 @@ Adapt.adapt_structure(to, grid::VerticallyStretchedRectilinearGrid{FT, TX, TY, T
 
 
 
+all_x_nodes(::Type{Center}, grid::VerticallyStretchedRectilinearGrid) = grid.xᶜᵃᵃ
+all_x_nodes(::Type{Face}, grid::VerticallyStretchedRectilinearGrid) = grid.xᶠᵃᵃ
+all_y_nodes(::Type{Center}, grid::VerticallyStretchedRectilinearGrid) = grid.yᵃᶜᵃ
+all_y_nodes(::Type{Face}, grid::VerticallyStretchedRectilinearGrid) = grid.yᵃᶠᵃ
+all_z_nodes(::Type{Center}, grid::VerticallyStretchedRectilinearGrid) = grid.zᵃᵃᶜ
+all_z_nodes(::Type{Face}, grid::VerticallyStretchedRectilinearGrid) = grid.zᵃᵃᶠ
```

Comment:
👀

Reply:
That was [your suggestion](https://github.com/CliMA/Oceananigans.jl/issues/1466#issuecomment-799796938)!

Reply:
Okay, just got warned that I committed to the wrong branch by mistake! Sorry

Reply:
Oh it's no great crime, provided you're ok with 👀 .  Everyone is tempted to scope creep every once in a while... :-P